packages/@react-spectrum/tag/src/index.ts

```diff
@@ -10,6 +10,5 @@
  * governing permissions and limitations under the License.
  */
 
-/// <reference types="css-module-types" />
-
-export * from './TagGroup';
+export {TagGroup} from './TagGroup';
+export {Item} from '@react-stately/collections';
```

Comment:
In the storybook story would you mind changing the import of Item from stately to here?